stroeer/page/article/v1/article_page.proto

```diff
@@ -17,7 +17,7 @@ message ArticlePage {
 
   // Additional related editorial articles. Articles which are not published
```

Comment:
Multiple sources possible now, not just `editorial`

Reply:
aber es sind ja trotzdem editorial articles, auch wenn die id von der team data machine vorgeschlagen wurde.